lib/apiservers/engine/backends/container.go

```diff
@@ -1521,17 +1531,19 @@ func portInformation(t *models.ContainerInfo, ips []netlink.Addr) []types.Port {
 	// (works with both IPv4 and IPv6 addresses)
 	var ports []types.Port
 
-	container := cache.ContainerCache().GetContainer(*t.ContainerConfig.ContainerID)
-	if container == nil {
-		log.Errorf("Could not find container with ID %s", *t.ContainerConfig.ContainerID)
+	cid := t.ContainerConfig.ContainerID
+	c := cache.ContainerCache().GetContainer(cid)
```

Comment:
+1.  I really hate go programmer's habit of overloading one line with a bazillion operations.  They need to remember the compiler doesn't care, and readability is more important.